### DISCUSSION TITLE
Converting GAMS file to XLSX

### DISCUSSION THREAD

**Discussion by saxenasatyam** [magpiemodel/magpie#240] [Q&A] [answered]
Dear Team,

How to convert my **fulldata.gams** in the output folder of the model run to **fulldata.xlsx**.

**Comment by tscheypidi** [comment, marked as answer]
`fulldata.gms` contains the model itself and cannot be converted to xls. There are tools out there to convert gdx-files (e.g. `fulldata.gdx`) to xls, which you could use (e.g. here: https://www.gams.com/33/docs/T_GDX2XLS.html).

However, MAgPIE uses a huge set of postprocessing routines all written in R which we would strongly recommend to use. `fulldata.gdx` only contains the raw, unprocessed output of the model. The processed data is produced from the R output routines (e.g. in package `magpie4` https://github.com/pik-piam/magpie4).
